espflash/Cargo.toml

```diff
@@ -50,6 +50,7 @@ log             = "0.4"
 md-5            = "0.10"
 miette          = "7.6"
 object          = "0.38"
+reed-solomon    = "0.2.1"
```

Comment:
This could be optional as it requires `serialport` feature if im not missing anything